cookbooks/aws-parallelcluster-common/attributes/common_ubuntu18.rb

```diff
@@ -0,0 +1,12 @@
+# Ubuntu 18 common attributes shared between multiple cookbooks
```

Comment:
If all the files common_* have the same content why not moving them into `cookbooks/aws-parallelcluster-common/attributes/common.rb`